### PR TITLE
updated README and CHANGELOG

### PR DESCRIPTION
**Pull request by yliaog**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# v31.0.0
+
+Kubernetes API Version: v1.31.1
```
Comment (tomplus) [inline CHANGELOG.md:3]: API changes from v1.31.1 are not included as code is generated for v1.31.0.

Comment (yliaog) [reply CHANGELOG.md]: Yes, the release tooling has a bug, fixed the CHANGELOG manually.

**Review by roycaihw — APPROVED** [review]
/lgtm
/approve

**Pull request merged**